base_custom_info/security/ir.model.access.csv

```diff
@@ -1,7 +1,5 @@
-"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
-"access_custom_info_template_user","custom.info.template.user","model_custom_info_template","base.group_user",1,0,0,0
-"access_custom_info_property_user","custom.info.template.line.user","model_custom_info_property","base.group_user",1,0,0,0
-"access_custom_info_value_user","custom.info.value.user","model_custom_info_value","base.group_user",1,0,0,0
-"access_custom_info_template_sale_manager","custom.info.template.salemanager","model_custom_info_template","base.group_system",1,1,1,1
-"access_custom_info_property_sale_manager","custom.info.template.line.salemanager","model_custom_info_property","base.group_system",1,1,1,1
-"access_custom_info_value_sale_manager","custom.info.value.salemanager","model_custom_info_value","base.group_system",1,1,1,1
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_template,Access custom info templates,model_custom_info_template,,1,1,1,1
```

Comment:
I realized that it makes no sense to have access rules for this model. Custom information is always linked to another model, that is the one with the access rules, so you will notice that I oveloaded each model's `check_access_rule` method to make them inherit the access rules of their linked model.

This way, if you can edit a product.template that has a custom.info.value, you can edit their values. If you cannot, you can't also edit their values. All is automatic and sensible.